feature/experimental/qos/ate_tests/qos_policy_config_test/qos_policy_config_test.go

```diff
@@ -81,6 +82,19 @@ func TestMain(m *testing.M) {
 //     - https://github.com/karimra/gnmic/blob/main/README.md
 //
 
+const (
+	dropProfile               = "DropProfile"
+	forwardingGroup           = "fc"
+	schedulerMap              = "smap"
+	sequeneId                 = 1
+	schedulerPriority         = 1
+	queueName                 = "0"
+	minThresholdPerecent      = 10
+	maxThresholdPercent       = 70
+	maxDropProbabilityPercent = 1
+	enableEcn                 = true
```

Comment:
Go initialism rule says this should be "enableECN"

Reply:
fixed